test/integration/admin_test.js

```diff
@@ -24,7 +24,8 @@ var RingpopClient = require('../../client.js');
 var testRingpopCluster = require('../lib/test-ringpop-cluster.js');
 
 testRingpopCluster({
-    size: 1
+    size: 1,
+    dontWaitForConvergence: true,
```

Comment:
can you flip this property around and make it read `waitForConvergence`? The double negatives "dont" and "false" make it confusing.